src/index.html

```diff
@@ -1,14 +1,42 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Londrina+Outline&display=swap" rel="stylesheet">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="container">
+      <div class="container__circle container__circle--color--1">
+        <div class="container__watch container__watch--number--1">
+          <div class="container__dot"></div>
+        </div>
+      </div>
+      <div class="container__circle container__circle--color--2"></div>
+      <div class="container__circle container__circle--color--3"></div>
+      <div class="container__circle container__circle--color--4"></div>
+      <div class="container__circle container__circle--color--5"></div>
+      <div class="container__circle container__circle--color--6"></div>
+      <div class="container__circle container__circle--color--7" >
```

Comment:
```suggestion
      <div class="container__circle container__circle--color-2"></div>
      <div class="container__circle container__circle--color-3"></div>
      <div class="container__circle container__circle--color-4"></div>
      <div class="container__circle container__circle--color-5"></div>
      <div class="container__circle container__circle--color-6"></div>
      <div class="container__circle container__circle--color-7" >
```
The number isn 't a modifier of color